application/ai/llm_control_service.py

```
@@ -253,6 +253,15 @@ def get_presets(self) -> List[LLMPreset]:
                 description='智谱 Anthropic-compatible 接口。模型名以智谱文档为准。',
                 tags=['domestic', 'preset'],
             ),
+            LLMPreset(
+                key='minimax-anthropic',
+                label='MiniMax / Anthropic 兼容',
+                protocol='anthropic',
+                default_base_url='https://api.minimaxi.com/anthropic',
+                default_model='MiniMax-M2.7',
+                description='MiniMax Anthropic-compatible 接口；默认使用 MiniMax-M2.7，也可改为 MiniMax-M2.7-highspeed 等官方模型 ID。',
```

Ruff (0.15.13)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)